src/modules/info-display/state/info-display-slice.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { RootState } from '@/store'
+
+interface DisplayState {
+  message: string
+}
+
+const initialState: DisplayState = {
+  message: '',
+}
+
+const displaySlice = createSlice({
+  name: 'display',
+  initialState,
+  reducers: {
+    setMessage(state, action: PayloadAction<DisplayState>) {
```

Comment:
`PayloadAction<string>` because you will pass only message to this action, you don't need other data.